benchmarks/dynamo/genai_kernels/benchmark.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+"""
+Benchmark runner for various kernel implementations.
+
+This script provides a command-line interface to run benchmarks for different
+kernel implementations including CrossEntropy, Softmax, RMSNorm, and LayerNorm
+kernels in both forward and backward directions.
+"""
+
+import argparse
+import sys
+
+from kernels import (
+    BenchmarkKernel,
+    CrossEntropyBackward,
+    CrossEntropyForward,
+    LayerNormBackward,
+    LayerNormForward,
+    RMSNormBackward,
+    RMSNormForward,
+    SoftmaxBackward,
+    SoftmaxForward,
+)
+
+import torch
+
+
+torch._dynamo.config.automatic_dynamic_shapes = False
+# Needed since changing args to function causes recompiles
+torch._dynamo.config.recompile_limit = 1000000
```

Comment:
maybe we can call `torch._dynamo.reset()` after each benchmark? not sure if it has the same effect